shared/package.json

```diff
@@ -3,6 +3,6 @@
 	"version": "0.1.0",
 	"private": true,
 	"devDependencies": {
-		"@camberi/firecms": "2.0.0-alpha.30"
+		"@camberi/firecms": "2.0.0-alpha.37"
```

Comment:
I think it would be nice if the shared package doesn't have a dependency to the firecms.

What do you think of the approach I had in https://github.com/socialincome-san/public/pull/86/files#diff-eb15213d223e369551834a6ab5f0831a9a4484dfe9a44dabb2e29f91898f84ac where I copied the EntityReference implementation into the shared package? It's the only dependency we use, and then we can get rid of this one.

Makes the shared and therefore functions package much slimmer and faster.

Reply:
But isn't that a problem if the dependent object changes..? Should we define it as a `peerDependency` instead?

Reply:
Hmm, but isn't that a problem when the dependent object changes? If it's just a `devDependency`, I don't see the problem. It will be in your `node_modules` folder anyway because it's being used in another project.

Reply:
Fair enough. Feels conceptually wrong to me that the functions have a compilation dependency to the firecms, but we can still iterate over it later.

```
npm ERR! Error: Missing script: "functions:test"

To see a list of scripts, run:
  npm run 
npm ERR!   in workspace: @socialincome/admin@0.1.0 
npm ERR!   at location: /home/node/admin 
npm ERR! Missing script: functions:test

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.user/.npm/_logs/2022-10-08T15_06_34_214Z-debug-0.log
make: *** [functions-test] Error 1

~/code/public restructure-functions *1 ?2 ❯ make functions-test   
```
should that work?

Reply:
Sorry my bad cached container.